etc/testing/circle/start-minikube.sh

```diff
@@ -11,6 +11,7 @@ minikube_args=(
   "--kubernetes-version=${VERSION}"
   "--cpus=7"
   "--memory=12Gi"
+  "--wait=all"
```

Comment:
why is `wait` necessary here if we're already waiting for docker below?

Reply:
just an extra layer of protection. this tells minikube start to [wait for Kubernetes components to verify and wait for after starting a cluster. ](https://minikube.sigs.k8s.io/docs/commands/start/)

Reply:
just an extra layer of protection. this tells minikube start to [wait for Kubernetes components to verify and wait for after starting a cluster. ](https://minikube.sigs.k8s.io/docs/commands/start/)